core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/columns/constructors.kt

```diff
@@ -167,44 +169,94 @@ internal fun <TD, T : DataFrame<TD>, C> Selector<T, ColumnSet<C>>.toColumns(
     }
 
 @JvmName("toColumnSetForPivot")
-internal fun <T, C> PivotColumnsSelector<T, C>.toColumns(): ColumnSet<C> = toColumns {
+internal fun <T, C> PivotColumnsSelector<T, C>.toColumnSet(): ColumnSet<C> = toColumnSet {
```

Comment:
While renaming these is good, we need to move them outside the impl package and make them public, otherwise users cannot use the @Deprecation with ReplaceWith correctly